crates/echo-wasm-bindings/README.md

```
@@ -0,0 +1,16 @@
+<!-- SPDX-License-Identifier: Apache-2.0 OR MIND-UCAL-1.0 -->
+<!-- © James Ross Ω FLYING•ROBOTS <https://github.com/flyingrobots> -->
+# echo-wasm-bindings
+
+Minimal WASM bindings shim for Echo/JITOS living specs. Provides `DemoKernel` with add/set/connect/delete and rewrite history, using `echo-wasm-abi` DTOs. Exports wasm-bindgen-friendly methods when built with `--features wasm`.
+
+## Dev
+- Native tests: `cargo test -p echo-wasm-bindings`
+- WASM build (example): `wasm-pack build --target web -F wasm` or via `trunk` when wired into spec pages.
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)